.github/workflows/collector-slim.yml

```
@@ -20,44 +20,69 @@ env:
   RHACS_ENG_IMAGE: quay.io/rhacs-eng/collector:${{ inputs.collector-tag }}
 
 jobs:
+  set-environment:
+    name: Set environment variables
+    runs-on: ubuntu-latest
+    outputs:
+      build-builder-image: ${{ steps.set-env.outputs.build-builder-image }}
+      collector-builder-tag: ${{ steps.set-env.outputs.collector-builder-tag }}
+
+    steps:
+      - name: Checks PR, main and release branches
+        id: set-env
+        run: |
+
+          if [[ "${{ github.event_name }}" != 'pull_request' ]]; then
+            echo 'build-builder-image=true' >> "$GITHUB_OUTPUT"
+            echo "collector-builder-tag=${DEFAULT_BUILDER_TAG}" >> "$GITHUB_OUTPUT"
+
+          elif [[ "${{ github.event_name }}" == 'pull_request' ]]; then
```

I believe we can do `else` directly here, since it will either not be `pull_request` and enter the first code block or be `pull_request` and enter this condition.
```suggestion
          else
```

---

Cool. I initially had kept it simply as `else` but thought of using an explicit condition. Will fix.